app_vue/src/layout/navigationBar.vue

```diff
@@ -66,23 +69,29 @@
         <v-divider class="mt-3 mb-3"></v-divider>
 
         <div>
+          <img v-show="state.expand" src="@/assets/WAV_LOGO.svg" alt="WAV Logo" class="wav-logo">
           <v-list-item class="navbar-links__item user">
-            <span>John Doe</span>
-            <span>jdoe@acme.com</span>
+            <span>Peter Uppal</span>
+            <span>peteru@wavsmart.com</span>
           </v-list-item>
           <v-list-item class="navbar-links__item" link href="/" rounded>
-            <vue-feather type="user"></vue-feather>
-            Profile
+            <vue-feather type="users" />
+            Account
           </v-list-item>
         </div>
 
         <v-divider class="mt-3 mb-3"></v-divider>
 
-        <v-list-item class="navbar-links__item" link href="/" rounded>
+      </v-list>
+      
+      <div v-show="state.expand"  id="bottom-section">
```

Comment:
i believe i can show you a nice trick with flexbox so that we don't have to use position:absolute method here to ensure the item is always on the bottom!

additionally i'm noticing an alignment issue with the sign out button here that would get resolved using the flexbox method as well:
<img width="151" alt="image" src="https://github.com/button-inc/iot-system-prototype/assets/7142197/dc16390b-9ab7-4642-8c7a-65e063903740">

Reply:
also done with my change!